demo/angular-grid.html

```diff
@@ -1,11 +1,11 @@
 <vaadin-grid #grid (selected-items-changed)="onSelect()">
   <table>
     <colgroup>
-      <col width=80 header-text="" name="user.picture.thumbnail">
-      <col header-text="Gender" name="user.gender">
-      <col header-text="First name" name="user.name.first">
-      <col header-text="Last name" name="user.name.last">
-      <col header-text="Email" name="user.email">
+      <col width=80 name="user.picture.thumbnail">
+      <col name="user.gender">
+      <col name="user.name.first">
+      <col name="user.name.last">
+      <col name="user.email">
```

Comment:
Thinking if we should define `<thead>` explicitly here. We probably don't want to have the `thumbnail` header and use `First Name` + `Last Name` even after the formatter is done?